pandas/tests/reshape/test_concat.py

```diff
@@ -1243,6 +1243,39 @@ def test_concat_dict(self):
         expected = concat([frames[k] for k in keys], keys=keys)
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "mapping",
+        [
+            pytest.param("mapping", id="Non dict subclass of collections.abc.Mapping"),
```

Comment:
since the parameters are strings, the strings will be the test ids, so no need to add these.